Cython/Compiler/ExprNodes.py

```diff
@@ -2176,6 +2176,7 @@ def nogil_check(self, env):
     gil_message = "Accessing Python global or builtin"
 
     def analyse_entry(self, env, is_target=False):
+        self.is_target = is_target
```

Comment:
I know, I suggested something like this, but I wonder if this isn't cleaner: https://github.com/cython/cython/pull/4281

Reply:
I pushed the changes to the master branch.